benchmark/catalog/catalog_benchmark.cpp

```diff
@@ -331,6 +331,11 @@ BENCHMARK_DEFINE_F(CatalogBenchmark, GetIndexObjects)(benchmark::State &state) {
   state.SetItemsProcessed(num_indexes * state.iterations());
 }
 
+// ----------------------------------------------------------------------------
```

Comment:
Not a fan of bringing this style of block comments over from Peloton but not a dealbreaker for me. Especially since it looks like they're made for 80-character lines (we use 120).